README.md

```diff
@@ -203,6 +203,50 @@ the signing key (the loading process will have to be done with other tools and w
 you to provivde your parent key password) into the TPM and referencing its handle in the command 
 you want to call with the credential helper. 
 
+#### Password-Encrypted Private Keys
+You can pass a password-encrypted private key to the credential helper for signing the request. The credential helper supports two formats of PKCS#8 private key files:
+- Unencrypted: `-----BEGIN PRIVATE KEY-----`
+- Password-encrypted: `-----BEGIN ENCRYPTED PRIVATE KEY-----` (using PBES2)
+
+To encrypt an unencrypted private key stored on disk, use a tool of your choice. The following example uses OpenSSL in a Bash environment:
```

Comment:
I would rephrase. 

Suggestion: "To encrypt a plaintext private key stored on disk, you can use `openssl`:"